src/themes/index.js

```diff
@@ -1,3 +1,3 @@
-import * as standard from './default';
+import * as sumup from './sumup';
 
-export { standard };
+export { sumup };
```

Comment:
I'd still export the `standard` here to keep the backward compatibility for while

Reply:
I would also be more in favor of calling it 'standard' and just renaming `default.js` to `standard.js` to be honest :D because we may have more sumup themes for different apps depending on their audience.